Deprecated CSS `-moz-user-input` property is removed (#37644)
mdn/content: files/en-us/mozilla/firefox/releases/135/index.md

#### Removals

> - The `-moz-user-input` CSS property allowed you to disable input fields. The property is now deprecated and has been removed ([Firefox bug 1935198](https://bugzil.la/1935198)).

Reviewer: I don't know if you want to link to that?

```suggestion
- The [`-moz-user-input`](/en-US/docs/Web/CSS/-moz-user-input) CSS property allowed you to disable input fields. The property is now deprecated and has been removed ([Firefox bug 1935198](https://bugzil.la/1935198)).
```

> - The -moz-user-input CSS property has been removed. This property has been deprecated and no longer allows the disabling of input fields [Firefox bug 1935198](https://bugzil.la/1935198).

Reviewer: ```suggestion
- The `-moz-user-input` CSS property allowed you to disable input fields. The property is now deprecated and has been removed ([Firefox bug 1935198](https://bugzil.la/1935198)).
```
